farmhelper-mod/src/main/java/com/jelly/farmhelper/features/Autosell.java

```diff
@@ -241,7 +241,7 @@ private static boolean shouldSell(ItemStack itemStack) {
             name.startsWith("Red Mushroom") || name.startsWith("Enchanted Red Mushroom") || name.startsWith("Red Mushroom Block") || name.startsWith("Red Enchanted Mushroom Block") ||
             name.startsWith("Nether Wart") || name.startsWith("Enchanted Nether Wart") || name.startsWith("Mutant Nether Wart") ||
             name.startsWith("Sugar Cane") || name.startsWith("Enchanted Sugar") || name.startsWith("Enchanted Sugar Cane") ||
-            name.startsWith("Cropie") || name.startsWith("Squash") || name.startsWith("Fermento") ||
+            name.startsWith("Cropie") || name.startsWith("Squash") || name.startsWith("Fermento") || name.startsWith("Burrowing") ||
```

Comment:
This method is for checking if the item should be sold to npc. We don't want to sell Burrowing Spored to npc, because the value is 1 coin.